src/main/resources/application-clint.yml

```diff
@@ -0,0 +1,144 @@
+#promregator.metrics.internal: true
+
+
+cf:
+  username: svc-ga-mon
+  password: kocriW6brOW1phiyLxun3FicHe6wEphu
```

Comment:
Please change your password now :)

Reply:
Good grief. How did I overlook this...

Rolling password...